packages/atlas/src/hooks/useSegmentAnalytics.ts

```diff
@@ -33,12 +33,9 @@ export const useSegmentAnalytics = () => {
 
   const playbackEventsQueue = useRef<{ type: playbackEventType; params: videoPlaybackParams }[]>([])
 
-  const identifyUser = useCallback(
-    (email = 'no data') => {
-      analytics.identify(email, { email })
-    },
-    [analytics]
-  )
+  const identifyUser = (email = 'no data') => {
+    analytics.identify(email, { email })
+  }
```

Comment:
@attemka did you still intend to remove the `useCallback` or was it just for the debugging ?